app/src/main/java/org/mozilla/fenix/compose/Favicon.kt

```diff
@@ -85,7 +86,7 @@ private fun FaviconPlaceholder(size: Dp) {
 @Composable
 @Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
 private fun FaviconPreview() {
-    FirefoxTheme {
+    FirefoxTheme(theme = Theme.getTheme(isPrivate = false)) {
```

Comment:
was setting it manually here to get the previews to build?

Reply:
Yes because preview doesn't know anything about `LocalContext.current.settings()` which is `components.settings`